src/styles/utils/mixin.scss

```diff
@@ -0,0 +1,23 @@
+@mixin number-day {
+  @for $cell from 1 through 31 {
+    & :nth-child(#{$cell})::before {
+      content: '#{$cell}';
+    }
+  }
+}
+
+@mixin weak-day-lentgh {
+  @each $weak-day, $index in $start-day {
+    &--start-day-#{$weak-day} :nth-child(1) {
+      margin-left: $cellSize * $index + $index;
```

Comment:
The calculation for 'margin-left' does not include units. It should be something like `margin-left: ($cellSize * $index) + ($gap * $index);`, where `$gap` is the space between cells and should be defined in your variables.